frontend/src/components/gradebook/student.tsx

```diff
@@ -1,16 +1,30 @@
 import React, { ReactElement } from "react";
 import { StudentInfo } from "../../types";
 
-const Student = (props: StudentInfo): ReactElement => {
+interface Props {
+  student: StudentInfo;
+  isExpanded: boolean[];
+}
+
+const Student = ({ student, isExpanded }: Props): ReactElement => {
   return (
     <tr className="student">
-      <td>{props.name}</td>
+      <td>{student.name}</td>
 
-      <td>{props.total}</td>
+      <td className="align-right">{student.total.toFixed(2)}</td>
 
-      {props.sessions.map((session, index: number) => (
-        <td key={index}>{session}</td>
-      ))}
+      {student.sessions.map((session, sIndex) => [
+        <td key={sIndex} className="align-right">
+          {session.toFixed(2)}
+        </td>,
+        isExpanded[sIndex]
+          ? student.questions[sIndex].map((question, qIndex) => (
```

Comment:
The StudentInfo type needs to be updated to include questions. Currently this does not build

Reply:
:+1: